sdk/storage/Azure.Storage.Blobs/src/Azure.Storage.Blobs.csproj

```diff
@@ -1,12 +1,12 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>$(RequiredTargetFrameworks);netstandard2.1</TargetFrameworks>
   </PropertyGroup>
   <PropertyGroup>
     <AssemblyTitle>Microsoft Azure.Storage.Blobs client library</AssemblyTitle>
     <Version>12.13.0-beta.1</Version>
     <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
-    <ApiCompatVersion>12.12.0</ApiCompatVersion>
+    <ApiCompatVersion Condition="'$(TargetFramework)' != 'netstandard2.1'">12.12.0</ApiCompatVersion>
```

Comment:
Temporary workaround from AzureSDK until an initial version with a netstandard2.1 target is on NuGet.